### DISCUSSION TITLE
Scalar constant multiplying by a quadvector

### DISCUSSION THREAD

**Discussion by sirlluis** [FeynCalc/feyncalc#251] [Q&A]
How to prevent Feyncalc from interpreting a scalar constant as a 4-vector by multiplying that constant by a 4-vector, i.e.

Let p some 4-vector and x some scalar. Feyncalc compute
$xP=\bar{xP}$
instead
$xP=x\bar{P}$

**Comment by vsht** [comment]
You need FCVariable

https://feyncalc.github.io/FeynCalcBookDev/FCVariable.html

Am 03.03.24 um 23:01 schrieb Luis Gerardo Ramirez Archundia:
